src/kbc_modules/eaa_kbc/mod.rs

```diff
@@ -15,6 +15,12 @@ pub mod protocol;
 pub mod rats_tls;
 use protocol::*;
 
+const DEFAULT_RECV_SIZE: usize = 4096;
```

Comment:
- ooi, why this value?
- Should it be configurable (the fact the name contains "default" suggests it could be overridden somehow)?
- Please add the units to the name of this `const` for clarity (`_BYTES`?)

Reply:
This is a default receiving buffer size determined after testing. It is used for some regular information interaction (such as version query). It can be overwritten. When calling it, the caller can decide whether to use this default size. It will give priority to the specified receiving buffer size.